compute/oslogin/physical_security_key_ssh.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```

Comment:
Should there also be a test file? `compute/oslogin/physical_security_key_ssh_test.py`